windows/src/engine/kmcomapi/com/packages/keymanpackagefile.pas

```diff
@@ -167,22 +167,31 @@ function TKeymanPackageFile.Get_ID: WideString;
 end;
 
 procedure TKeymanPackageFile.Install(Force: WordBool);
+var
+  o: TKPInstallPackageOptions;
 begin
   with TKPInstallPackage.Create(Context) do
   try
-    Execute(FFileName, Force, True);
+    o := [ipLegacyRegisterAndInstallProfiles];
+    if Force then
+      Include(o, ipForce);
```

Comment:
Where does `ipForce` come from?

Reply:
`ipForce` is defined in the enumeration in kpinstallpackage.pas:

```
TKPInstallPackageOptions = set of (ipForce, ipLegacyRegisterAndInstallProfiles);
```